pkg/nfd-master/nfd-master.go

```diff
@@ -78,7 +80,7 @@ type NFDConfig struct {
 	ResyncPeriod      utils.DurationVal
 	LeaderElection    LeaderElectionConfig
 	NfdApiParallelism int
-	Klog              map[string]string
+	KlogOptions       klogutils.KlogConfigOpts
```

Comment:
I think we cannot change the name of the field as that is the one used in the actual config file (json/yaml).

```suggestion
	Klog              klogutils.KlogConfigOpts
```